src/infer.ts

```diff
@@ -636,6 +636,9 @@ function inferExpressionNullability(
     // EXISTS (subquery) never returns NULL
     existsOp: () => anyTE(false),
 
+    // ANY (subquery) never returns NULL
+    anyOp: () => anyTE(true),
+
```

Comment:
This is not true. According to the docs, `lhs op any (...)` is null with the same rules as `lhs in (...)`.